apps/studio/components/interfaces/Realtime/Inspector/RealtimeFilterPopover/TableSelector.tsx

```diff
@@ -40,114 +41,141 @@ const TableSelector = ({
   onSelectTable,
 }: TableSelectorProps) => {
   const [open, setOpen] = useState(false)
+  const [initiallyLoaded, setinitiallyLoaded] = useState(false)
```

Comment:
minor nit hahaha
set_I_nitiallyLoaded
```suggestion
  const [initiallyLoaded, setInitiallyLoaded] = useState(false)
```